index.mjs

```diff
@@ -0,0 +1,4 @@
+import EventEmitter from './index.js'
+
+export { EventEmitter }
+export default EventEmitter
```

Comment:
```suggestion
export default EventEmitter

```

Reply:
sure, do you want to add `.editorconfig` for that? I do that in my repos.


```
root = true

[*]
charset = utf-8
end_of_line = lf
insert_final_newline = true
indent_size = 2
indent_style = space
trim_trailing_whitespace = true
```

Reply:
No thanks, it can be enforced with linting but this is outside the scope of the PR.

Reply:
👌